Aberrus/Locales/zhTW.lua

```diff
@@ -38,19 +38,19 @@ L = BigWigs:NewBossLocale("Assault of the Zaqali", "zhTW")
 if L then
 	-- These are in-game emotes and need to match the text shown in-game
 	-- You should also replace the comment (--) with the full emote as it shows in-game
-	--L.zaqali_aide_north_emote_trigger = "northern battlement" -- |TInterface\\ICONS\\Ability_Hunter_KillCommand.blp:20|t Commanders ascend the northern battlement!
-	--L.zaqali_aide_south_emote_trigger = "southern battlement" -- |TInterface\\ICONS\\Ability_Hunter_KillCommand.blp:20|t Commanders ascend the southern battlement!
+	L.zaqali_aide_north_emote_trigger = "北方城垛" -- |TInterface\\ICONS\\Ability_Hunter_KillCommand.blp:20|t Commanders ascend the northern battlement!
```

Comment:
As stated in the comments above these lines, you need to replace the comment with the full translation from in-game so we have a reference if we need to change it in future.